CHANGES.rst

```diff
@@ -74,10 +74,8 @@ New Features
     browsing capabilities. To write out to this format, use
     ``Table.write('filename.html', format='jsviewer')``. [#2875]
 
-  - A ``quantity`` property and ``to`` method were added to ``Table``
```

Comment:
It seems it was removed in f2c15671ad02e4343e1b9ce0fbc7f874ff4921b1  This should definitely be fixed.
It can be fixed by the following process:

1. Run `$ git rebase -i master`
2. In the text file that comes up, mark the commit f2c15671ad02e4343e1b9ce0fbc7f874ff4921b1 as "edit" (that is, replace the word "pick" with "edit")
3. Save the file; exit.  The rebase will proceed until f2c15671ad02e4343e1b9ce0fbc7f874ff4921b1 and then pause.
4. Run `$ git reset HEAD^`--this will basically undo that commit.  Typing `$ git status` will show the changes in that commit as though it were not saved yet.
5. You can edit the commit manually to restore the removed line, or run `$ git checkout --patch` to selectively undo edits (keep the added line, discard the change that removes the accidentally removed lines).
6. Save the new commit (with its original commit message) by running `$ git commit -C ORIG_HEAD`
7. Complete the rebase: `$ git rebase --continue`
8. Push the new changeset: `$ git push --force origin new-jsviewer-css`
9. :beers: